docusaurus.config.js

```diff
@@ -4,6 +4,22 @@ const fetch = require('node-fetch');
 
 const VERSIONS_JSON = require('./versions.json');
 
+/**
+ * Old versions of the Ionic Docs are archived so
+ * that we do not need to re-build it every time we deploy.
+ * Building a large number of docs sites at once can cause
+ * out of memory issues, so archiving old docs sites
+ * allow us to keep memory usage and build times low.
+ */
+const ARCHIVED_VERSIONS_JSON = require('./versionsArchived.json');
```

Comment:
Can we also add a note that this JSON file is only for versions that live in the site? Else we might be questioning why v4 and v3 are not included.

Reply:
Good idea! Added in https://github.com/ionic-team/ionic-docs/pull/3482/commits/914bc7ef132de0201831d39f04619d10ef98030e